lib/blip.js

```diff
@@ -1,3 +1,8 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
```

Comment:
Not errors.. this bit of code is responsible for much drama.. Best move along :-P

(I'm going to delete this soon, this PR is already big enough)